src/fpm/utils.py

```diff
@@ -33,28 +34,40 @@ def save_file(output_path, file_name, contents):
     if ext in [".yaml"]:
         with open(output_file, "w") as f:
             yaml.dump(contents, f, default_flow_style=None)
+    elif ext in [".pgm"]:
```

Comment:
shouldn't these be just string comparisons?

Reply:
yeah, I think when I wrote this originally I expected more file formats would reuse the same if blocks.